example/app/storage/page.tsx

```diff
@@ -1,19 +1,26 @@
 'use client'
+
 import { useEffect } from 'react'
-import { useFile, useFileDelete, useFileDownload, useFileUpload } from 'react-appwrite/storage'
+import { useFile, useBucket, useFileDelete, useFileDownload, useFileUpload, useFileView, useFilePreview } from 'react-appwrite/storage'
+import Image from 'next/image'
 
 export default function StoragePage() {
   const upload = useFileUpload()
   const { data } = useFile('test', 'test')
   const deleteFile = useFileDelete()
   // const testPreview = useFilePreview('test', 'test')
   const download = useFileDownload()
+  const fileView = useFileView()
+  const { data: uploadedImages } = useBucket("test")
+  const { data: preview } = useFilePreview('test', 'test', { height: 100, width: 100})
 
   useEffect(() => {
     download.mutate({ bucketId: 'test', fileId: 'test' })
   }, [])
 
-  console.log({ download })
+  useEffect(() => {
+    fileView.mutate({ bucketId: 'test', fileId: 'test' })
+  }, [])
```

Comment:
This is why we should be avoiding mutations. Developers shouldn't have to create a `useEffect` just to start fetching their data. It goes against the purpose of the project.

To be clear, this is not your fault @dishwasher-detergent, this is completely on me for misusing the mutations in the first place.